src/tasks.md

```diff
@@ -0,0 +1,263 @@
+# Common Tasks
+
+This page lists a set of tasks that can be performed after you [install Kubewarden](/quick-start.html#install) in your Kubernetes cluster.
+
+Each task can be done separately; however, if you're not familiar with Kubewarden, or Kubernetes policies in general, we recommend that you follow the tasks below in sequential order.
+
+## Test Policies
+
+Kubewarden has two main tools to help you find policies and test them locally:
+
+- [Kubewarden Policy Hub](https://hub.kubewarden.io/)
+- [`kwctl`](https://github.com/kubewarden/kwctl) CLI tool
+
+### Kubewarden Policy Hub
+
+The Kubewarden Policy Hub hosts a set of pre-made policies that are created by the Kubewarden developers. These policies are based on the current set of [Kubernetes Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+
+As shown in the picture below, once you find the policy to be tested, you can copy the registry path<sup>1</sup> or download<sup>2</sup> the `WASM` binary containing the policy and additional metadata:
+
+![Kubewarden Policy Hub](/images/tasks-policy-hub.png)
+
+Once you have the policy `WASM` binary or the registry path, you can test it with `kwctl`.
+
+### `kwctl` CLI tool
+
+`kwctl` is a Command Line Interface (CLI) tool that will allow both the policy authors and the cluster administrators to test policies before they are applied to the Kubernetes cluster.
+
+The user experience (UX) of this tool is intended to be as easy and intuitive as possible. Think of it as the `docker` CLI tool if you were working with containers.
```

Comment:
```suggestion
The user experience (UX) of this tool is intended to be as the `docker` CLI tool.
```

I think it's not needed to explain our intentions, we do intend for everything to be as easy and intuitive as possible.

Reply:
Rephrased it based on your suggestion